ports/rtaudio/vcpkg.json

```
@@ -1,7 +1,13 @@
 {
   "name": "rtaudio",
-  "version-date": "2021-01-25",
+  "version-date": "2021-04-30",
   "description": "A set of C++ classes that provide a common API for realtime audio input/output across Linux (native ALSA, JACK, PulseAudio and OSS), Macintosh OS X (CoreAudio and JACK), and Windows (DirectSound, ASIO and WASAPI) operating systems.",
   "homepage": "https://github.com/thestk/rtaudio",
-  "supports": "!uwp"
+  "supports": "!uwp",
+  "features": {
+    "asio": {
+      "description": "Build with ASIO backend"
```

```suggestion
      "description": "Build with ASIO backend",
```
You can also use `./vcpkg format-manifest --all` to format